src/web/arr/trove/chart.arr

```diff
@@ -398,10 +406,10 @@ data DataSeries:
     method show-outliers(self, show):
       self.constr()(self.obj.{show-outliers: show})
     end,
-    method min(self, min):
+    method min(self :: DataSeries, min :: Number) -> Option<Number>:
```

Comment:
IIUC (it's been long time I last touched this code), the return type should be a `DataSeries`. But you can also leave it out, since most other methods also return a `DataSeries` and they didn't have the return type annotation.

I'm surprised at the conversation in Slack and this PR. Why don't you get a syntax error when you have a syntactically invalid program? And why doesn't this PR error to you? Are you sure that you have the right environment for testing (say, not testing a wrong branch or something like that)?